components/centraldashboard/public/components/logout-button.js

```diff
@@ -16,7 +16,7 @@ export class LogoutButton extends PolymerElement {
             <a href$="{{logoutUrl}}" on-tap="logout">
                 <paper-button id="logout-button">
                     <iron-icon icon="kubeflow:logout" 
-                               title="Logout">
+                               title="{{localize(landingPage.btnLogout")}}">
```

Comment:
I think this should be `title="{{localize('landingPage.btnLogout')}}"`

Reply:
oh yeah there is too many of them

Reply:
![image](https://github.com/user-attachments/assets/4b894a78-7283-4ab4-a4f0-4fc22f63a5f9)